START.md

```diff
@@ -221,6 +223,18 @@ Usage of ./autospotting:
         or if you want to set your bid price to be higher than the on demand
         price to reduce the chances that your spot instances will be terminated.
         (default 1)
+  -spot_price_multiplier float
+        Multiplier for the spot price. Placing bids a fraction of a cent over the
+        current price would aggressively optimize for cost savings since the last
+        runtime hour before spot termination is free of charge.
+        (default 1.1)
+
+  -spot_price_multiplier_flag bool
```

Comment:
I would prefer to have this a multi-option flag instead of a boolean, perhaps named something like 'bidding_policy', and accepting string values such as 'normal' and 'aggressive' and document them in detail. This way we can add more policies later on.

Reply:
The idea would be to have a "policy type" and a tag associated with the configuration of this specific policy, is that correct @cristim ?

Reply:
Yes